R/list-combine.R

```diff
@@ -22,6 +22,9 @@
 #'   same size (i.e. number of rows).
 #' @param name_repair One of `"unique"`, `"universal"`, or `"check_unique"`.
 #'   See [vctrs::vec_as_names()] for the meaning of these options.
+#' @param keep_empty  An optional logical. If `FALSE` (the default), then
```

Comment:
I think `keep_empty` is the wrong name if `double()` isn't promoted to `NA_real_` and kept. It's more like `keep_null`.

```r
> list_c(list(1, double(), 2))
[1] 1 2
> list_c(list(1, double(), 2), keep_empty = T)
[1] 1 2
```

Reply:
Compare with

```r
> tidyr::unchop(tibble::tibble(x = list(1, double(), 2)), x)
# A tibble: 2 × 1
      x
  <dbl>
1     1
2     2
> tidyr::unchop(tibble::tibble(x = list(1, double(), 2)), x, keep_empty = TRUE)
# A tibble: 3 × 1
      x
  <dbl>
1     1
2    NA
3     2
```